plugins/woocommerce-admin/client/marketplace/components/product-list-content/product-list-content.tsx

```diff
@@ -4,24 +4,37 @@
 import './product-list-content.scss';
 import ProductCard from '../product-card/product-card';
 import { Product } from '../product-list/types';
+import { appendURLParams } from '../../utils/functions';
+import { getAdminSetting } from '../../../utils/admin-settings';
 
 export default function ProductListContent( props: {
 	products: Product[];
 } ): JSX.Element {
-	const { products } = props;
+	const wccomHelperSettings = getAdminSetting( 'wccomHelper', {} );
+
 	return (
 		<div className="woocommerce-marketplace__product-list-content">
-			{ products.map( ( product ) => (
+			{ props.products.map( ( product ) => (
 				<ProductCard
 					key={ product.id }
 					type="classic"
 					product={ {
 						title: product.title,
 						icon: product.icon,
 						vendorName: product.vendorName,
-						vendorUrl: product.vendorUrl,
+						vendorUrl: appendURLParams( product.vendorUrl, [
+							[ 'utm_source', 'extensionsscreen' ],
+							[ 'utm_medium', 'product' ],
+							[ 'utm_campaign', 'wcaddons' ],
+							[ 'utm_content', 'devpartner' ],
+						] ),
```

Comment:
We should've had this in the test steps! But I've checked it on both discovery and browse screens, and it reproduces what we have in WooCommerce 8.0.

Reply:
> We should've had this in the test steps! But I've checked it on both discovery and browse screens, and it reproduces what we have in WooCommerce 8.0.

I have added to the testing steps, although a bit late now. Still good to have it in there. 

> I would've done the appendURLParams call separately, and just passed the result here, for better legibility. But that's another debatable stylistic choice.

Totally understandable. Since this is in `Array.map`, I didn't want to turn that into a multiline code block and keep it a single statement.